doc/releases/release-notes-3.6.rst

```diff
@@ -29,6 +29,10 @@ Architectures
 
 * ARC
 
+ * Extended support for ARCv3 features: enable HW prefetch and shared cache;
+ * Disabled TLS support for platforms with two or more register banks;
+ * Fixed unstable work of images built with MWDT on HW platforms (garbage in .device_states section).
```

Comment:
Fixed unstable work of application built with MetaWare toolchain for hardware platforms (garbage in .device_states section).